webview-ui/src/components/ChatPage/ChatPage.css

```
@@ -0,0 +1,1569 @@
+.chat-page-container {
+  width: 100%;
+  height: 100%;
+  container-type: inline-size;
+  container-name: chatPage;
```

Stylelint (17.7.0)

[error] Expected container name "chatPage" to be kebab-case (container-name-pattern)

(container-name-pattern)